test/WebJobs.Script.Tests.Integration/Host/ScriptHostManagerTests.cs

```diff
@@ -32,7 +32,7 @@ public ScriptHostManagerTests()
         public async Task UpdateFileAndRestart()
         {
             CancellationTokenSource cts = new CancellationTokenSource();
-            var fixture = new NodeEndToEndTests.TestFixture();
+            var fixture = new CSharpEndToEndTests.TestFixture();
```

Comment:
Not trivially, because `NodeEndToEndTests` is omitted when building for Mono. I could `#if` out `UpdateFileAndRestart`, I guess.